hydromt_fiat/fiat.py

```diff
@@ -417,9 +422,27 @@ def setup_exposure_buildings(
                 self.spatial_joins["additional_attributes"] = []
             self.spatial_joins["additional_attributes"].append(attrs) 
 
-
+        # Check for the required columns
         self.exposure.check_required_columns()
 
+        # Possibly reproject according to destination crs
+        src_crs = CRS.from_user_input(self.exposure.crs)
+        crs = None
+        if dst_crs is not None:
+            crs = CRS.from_user_input(dst_crs)
```

Comment:
you migth want to `try-except` invalid CRS user input here, but I'll leave that up to you

Reply:
Good idea. Will include that.